tests/lib/hardware/hardware_base.py

```diff
@@ -90,6 +89,13 @@ def public_key(self):
     def private_key(self):
         return self._private_key
 
+    def _get_working_dir(self):
+        os.makedirs(config.WORKSPACE_DIR)
+        return tempfile.mkdtemp(
```

Comment:
Do we still need a tempdir here? Given that we have the uuid which is unique already?

Reply:
good point, let me fix it